contrib/blueflood-docker-compose/docker-compose-cass-es.yml

```diff
@@ -0,0 +1,61 @@
+# using Docker-Compose Version 3
+version: '3'
+
+services:
+  # Configuration for seed cassandra node. Also, this is the only node
+  cassandra:
+    container_name: cassandra-node
+    image: cassandra:2.1
+    ports:
+        - "7000:7000"
+        - "7001:7001"
+        - "9042:9042"
+        - "9160:9160"
+        - "7199:7199"
+
+    # Exposing ports for inter cluster communication.
+    expose:
+      - 7000 # Intra-node communication
+      - 7001 # TLS intra-node communication
+      - 7199 # JMX
+      - 9042 # CQL
+      - 9160 # Thrift service
+
+    command: bash -c 'if [ -z "$$(ls -A /var/lib/cassandra/)" ] ; then sleep 0; fi && /docker-entrypoint.sh cassandra -f'
+
+    # Network for the nodes to communicate
+    networks:
+      - dc1ring
+
+    # Maps cassandra data to a local folder. This preserves data across
+    # container restarts. Note a folder 'cassdata' gets created locally in HOME directory.
+    volumes:
+      - ${HOME}/cassdata:/var/lib/cassandra
```

Comment:
An alternative is to let `docker-compose` manage retaining the volumes (until you do a `docker-compose down -v`) and use a named volume like

```
    - cassdata:/var/lib/cassandra
```

and then further down in declare a volumes section:

```
volumes:
    cassdata:
```

No need to change what you have, because there are advantages to what you've specified. Just food for thought.